puppet_forge.gemspec

```diff
@@ -20,16 +20,17 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 3.1.0'
 
+  spec.add_runtime_dependency "base64", '~> 0.2.0'
```

Comment:
There is no difference between `add_runtime_dependency` and `add_dependency`: https://guides.rubygems.org/specification-reference/#add_dependency. I don't like things having aliases, but that's Ruby for you.

Reply:
I added rubocop in #130 , to enforce a consistent style.